src/Store.ts

```diff
@@ -76,11 +77,15 @@ export type FormatAction = {
 } | {
     type: 'Set_Measurements',
     carouselMargin: number
+} | {
+    type: 'Set_Text_Format_Option',
+    messageTextFormat: string
```

Comment:
if it goes in `ConnectionState` I think you can piggyback on an existing action.